packages/syft/src/syft/node/node.py

```diff
@@ -611,60 +621,10 @@ def named(
         migrate: bool = False,
         in_memory_workers: bool = True,
     ) -> Self:
+        uid = UID.with_seed(name)
         name_hash = hashlib.sha256(name.encode("utf8")).digest()
-        name_hash_uuid = name_hash[0:16]
```

Comment:
this is much better

Reply:
Sorry didn't get you. Do you prefer the UID.with_seed()? or do you want me to revert to this name_hash[0:16] method?

Reply:
no this whole node.named method